src/actions/sessionActions.js

```diff
@@ -0,0 +1,25 @@
+import { SIGN_IN_START, SIGN_IN_SUCCESS, SIGN_IN_FAIL } from './actionTypes';
+import * as sessionService from '../services/session';
+
+export const signInStart = () => ({ type: SIGN_IN_START });
+
+export const signInSuccess = (user) => ({
```

Comment:
Remove unnecessary parenthesis around user, functions with implicit return that only have one parameter don't need them. Check for other instances.